src/lib/cache/SubscriptionCache.cpp

```diff
@@ -219,6 +219,30 @@ Subscription* SubscriptionCache::lookupById
 
 /* ****************************************************************************
 *
+* SubscriptionCache::lookupById - 
```

Comment:
The line `ADD_SUBDIRECTORY(src/lib/cache)` has been removed from CMakeFile.txt. However, it seems that not only we still having files under src/lib/cache but also this PR includes modification in these files. I'm missing something :)